File: 2058-concatenation-of-array/concatenation-of-array.java
class Solution {
    //Rehan Baig (gitHub:@raaay_haan)
    public int[] getConcatenation(int[] nums) {
        int [] ans = new int[2 * nums.length];
        for(int i = 0; i < nums.length; i++){
            ans[i] = nums[i];
            int size = i + nums.length;
            ans[size] = nums[i];
        }
        return ans;
    }
}

File: 2058-concatenation-of-array/concatenation-of-array.py
from typing import List
class Solution:
    #Rehan Baig 
    #gitHub:@raaay_haan
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        ans = [0] * (2*n)
        for i in range(n):
            ans[i] = nums[i]
            ans[i+n] = nums[i]
        return ans
        